library/public/image_c_api.h

```diff
@@ -0,0 +1,53 @@
+#ifndef F3D_C_API_H
+#define F3D_C_API_H
+
+#include "export.h" // Ensure F3D_EXPORT is defined
```

Comment:
Is this type of comment valid in C? Or should you do something like
```suggestion
#include "export.h" /* Ensure F3D_EXPORT is defined */
```